department-of-veterans-affairs/caseflow
# Fix storybook build

Resolves [APPEALS-24120](https://jira.devops.va.gov/browse/APPEALS-24120)

(branched from feature/APPEALS-4013)

# Description
Fixes the storybook build in deployed environments

## Acceptance Criteria
- [ ] Storybook does not display an error in deployed environments

## Testing Plan

1. Run `yarn build-storybook` from the client directory
2. Open the generated index.html and ensure no errors show up
